src/prepack-cli.js

```diff
@@ -274,6 +281,10 @@ function run(
       if (outputSourceMap) {
         fs.writeFileSync(outputSourceMap, serialized.map ? JSON.stringify(serialized.map) : "");
       }
+      if (heapGraphFile) {
+        invariant(serialized.heapGraph);
+        fs.writeFileSync(heapGraphFile, serialized.heapGraph);
```

Comment:
Since we are doing IO shouldn't there be some error checking?

Reply:
I am matching the pattern as other file I/O code here. 
writeFileSync() will throw exception while failing. The parent function's try...catch will handle this case.